crates/telio-traversal/src/endpoint_providers/stun.rs

```diff
@@ -229,6 +238,33 @@ impl<Wg: WireGuard, E: Backoff + 'static> EndpointProvider for StunEndpointProvi
         .await
         .unwrap_or(None)
     }
+
+    async fn pause(&self) {
+        let _ = task_exec!(&self.task, async move |s| {
+            if !s.is_battery_optimization_on {
+                return Ok(());
+            }
+            if s.stun_state != StunState::Paused {
+                s.servers = Vec::new();
```

Comment:
I believe that after resetting servers here - will not allow us to restore them on unpause, meaning that stun endpoint provider will be left stuck here: https://github.com/NordSecurity/libtelio/pull/168/files#diff-f52b9bdf9a9389d5ebb2456346414b982fc5fd532c9cf08e23284a430ff06003R555